File: Daily Challanges/2026-03-March/2026-03-19.js
function invertMatrix(matrix) {
  let values = new Set();
  for (let i = 0; i < matrix.length; i++) {
    for (let j = 0; j < matrix[0].length; j++) {
      values.add(matrix[i][j]);
    }
  }

  const [a, b] = values;
  for (let i = 0; i < matrix.length; i++) {
    for (let j = 0; j < matrix[0].length; j++) {
      matrix[i][j] = matrix[i][j] === a ? b : a;
    }
  }
  return matrix;
}

console.log(
  invertMatrix([
    ["a", "b"],
    ["a", "a"],
  ]),
);

File: Daily Challanges/2026-03-March/2026-03-19.py
def invert_matrix(matrix):
    values = set()
    for row in matrix:
        for elem in row:
            values.add(elem)
    vals = tuple(values)

    for i in range(len(matrix)):
        for j in range(len(matrix[0])):
            matrix[i][j] = vals[0] if matrix[i][j] == vals[1] else vals[1]
    return matrix


print(invert_matrix([["a", "b"], ["a", "a"]]))
